lib/process-services-cloud/src/lib/task/start-task/components/people-cloud/people-cloud.component.ts

```diff
@@ -187,9 +187,15 @@ export class PeopleCloudComponent implements OnInit, OnChanges {
                 this.invalidUsers.push(user);
             }
         });
+        validUsers =  this.removeDuplicatedUsers(validUsers);
         return validUsers;
     }
 
+    private removeDuplicatedUsers(users) {
+        return users.filter((user, index, self) =>
+                    index === self.findIndex((t) => t.id === user.id));
```

Comment:
single letter variable is a thing that we usually don't do they are bad for readability